tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/Configs.scala

```diff
@@ -482,10 +482,15 @@ object Configs {
             config.getBoolean("header")
           else
             false
+        val csvFields = if (config.hasPath("csv.fields"))
```

Comment:
Thanks for your contribution, please add an example in `nebula-java/tools/exchange/src/main/resources/application.conf` to show the way to config csvFields for other users.